airbyte-integrations/connectors/source-aircall/metadata.yaml

```diff
@@ -2,7 +2,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: 912eb6b7-a893-4a5b-b1c0-36ebbe2de8cd
-  dockerImageTag: 0.1.0
+  dockerImageTag: 0.2.0
```

Comment:
Please update the Dockerfile version too.

Reply:
done